### DISCUSSION TITLE
Traffic failures during DDoS

### DISCUSSION THREAD

**Discussion by mellfice** [akvorado/akvorado#591] [Q&A]
Hello

Noticed the following behavior akvorado during DDoS attacks. 

When there is a large DDoS attack (>10G), akvorado loses some data. On the graph it can be seen as a drop in traffic.

On the main page you can see that the amount of flow decreases. At some point the amount of flow becomes 0. 
Restart docker-compose helps.

Please advise if there is any way to fix this situation. May be some help to take action during DDoS. Or may I need some additional data?
 
Exporters: 6
BMP: enabled
Flows/s: 10-15k
Traffic: >100G
 
Thank you

![image](https://user-images.githubusercontent.com/73892787/227864459-e64f92cd-8de0-4bdd-a3ae-a1ffeb65559f.png)

**Comment by vincentbernat** [comment]
Check the troubleshooting section of the documentation. There are metrics to check if Akvorado is loosing packets.

**Comment by morhold** [comment]
it is possible that your equipment does not have time to sample during the attack if the sampling rate is set to a low value
